uninstall.php

```diff
@@ -100,36 +124,43 @@ public function uninstall() {
 			wp_delete_post( $id, true );
 		}
 
-		// Delete all what is related to languages and translations
+		// Delete all what is related to languages and translations.
 		$term_ids = array();
 		$tt_ids   = array();
 
-		foreach ( get_terms( array( 'taxonomy' => $pll_taxonomies, 'hide_empty' => false ) ) as $term ) {
+		$terms = get_terms(
+			array(
+				'taxonomy'   => $pll_taxonomies,
+				'hide_empty' => false,
+			)
+		);
+
+		foreach ( $terms as $term ) {
 			$term_ids[] = (int) $term->term_id;
-			$tt_ids[] = (int) $term->term_taxonomy_id;
+			$tt_ids[]   = (int) $term->term_taxonomy_id;
 		}
 
 		if ( ! empty( $term_ids ) ) {
 			$term_ids = array_unique( $term_ids );
-			$wpdb->query( "DELETE FROM {$wpdb->terms} WHERE term_id IN ( " . implode( ',', $term_ids ) . ' )' ); // PHPCS:ignore WordPress.DB.PreparedSQL.NotPrepared
-			$wpdb->query( "DELETE FROM {$wpdb->term_taxonomy} WHERE term_id IN ( " . implode( ',', $term_ids ) . ' )' ); // PHPCS:ignore WordPress.DB.PreparedSQL.NotPrepared
-			$wpdb->query( "DELETE FROM {$wpdb->termmeta} WHERE term_id IN ( " . implode( ',', $term_ids ) . " ) AND meta_key='_pll_strings_translations'" ); // PHPCS:ignore WordPress.DB.PreparedSQL.NotPrepared
+			$wpdb->query( $wpdb->prepare( sprintf( "DELETE FROM {$wpdb->terms} WHERE term_id IN (%s)", implode( ',', array_fill( 0, count( $term_ids ), '%d' ) ) ), $term_ids ) );
+			$wpdb->query( $wpdb->prepare( sprintf( "DELETE FROM {$wpdb->term_taxonomy} WHERE term_id IN (%s)", implode( ',', array_fill( 0, count( $term_ids ), '%d' ) ) ), $term_ids ) );
+			$wpdb->query( $wpdb->prepare( sprintf( "DELETE FROM {$wpdb->termmeta} WHERE term_id IN (%s) AND meta_key='_pll_strings_translations'", implode( ',', array_fill( 0, count( $term_ids ), '%d' ) ) ), $term_ids ) );
```

Comment:
```suggestion
		$wpdb->query(
			$wpdb->prepare(
			sprintf(
				"DELETE FROM {$wpdb->terms} WHERE term_id IN (%s)",
				implode( ',', array_fill( 0, count( $term_ids ), '%d' ) )
			),
			$term_ids
			)
		    );
    		    $wpdb->query(
			$wpdb->prepare(
				sprintf(
					"DELETE FROM {$wpdb->term_taxonomy} WHERE term_id IN (%s)",
					implode( ',', array_fill( 0, count( $term_ids ), '%d' ) )
				),
				$term_ids
			)
		 );
		$wpdb->query(
			$wpdb->prepare(
				sprintf(
					"DELETE FROM {$wpdb->termmeta} WHERE term_id IN (%s) AND meta_key='_pll_strings_translations'",
					implode( ',', array_fill( 0, count( $term_ids ), '%d' ) )
				),
				$term_ids
			)
		);
```
Suggesting some indentation for readability.